config/settings/common.py

```diff
@@ -169,7 +172,7 @@
                 'django.template.context_processors.static',
                 'django.template.context_processors.tz',
                 'django.contrib.messages.context_processors.messages',
-                # Your stuff: custom template context processors go here
+                'mrbelvedereci.users.context_processors.admin_url',
```

Comment:
why a context processor, and not just reverse it with ` {% url 'admin:index' %}`

Reply:
Good point!

Honestly, it's been a while since I used a context processor and that was the references I found when searching this use case online.  But, I was searching for how to get access to settings rather than the more generic: "How do a build a link to admin in a template?".  I'll change to the tag and remove the context processor.

In the future, I anticipate we could gain a lot from using context processors more though.  They're super easy.  This just isn't the right use case.